### DISCUSSION TITLE
manager/configuration-and-advanced-settings/How-to-set-up-Telegram-proxy-on-Hiddify/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4538] [Content Help کمک به اصلاح محتوا]
# manager/configuration-and-advanced-settings/How-to-set-up-Telegram-proxy-on-Hiddify/

The proxy manager platform.

https://hiddify.com/manager/configuration-and-advanced-settings/How-to-set-up-Telegram-proxy-on-Hiddify/?h=

**Comment by glory7sa** [comment]
MTProxybot

Incorrect secret value. It must contain 32 hex characters like 
00112233445566778899aabbccddeeff.

Where to get this code is not specified.
